src/bundle/Resources/public/scss/_summary-tile.scss

```diff
@@ -0,0 +1,79 @@
+.ibexa-summary-tile {
+    display: flex;
+    flex-direction: column;
+    padding: calculateRem(24px) calculateRem(24px);
+    gap: calculateRem(8px);
+    width: calculateRem(450px);
+    background: $ibexa-color-white;
+    border: calculateRem(1px) solid $ibexa-color-light-400;
+    border-radius: $ibexa-border-radius;
+
+    &__header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__icon-wrapper {
+        display: flex;
+        flex-shrink: 0;
+        align-items: center;
+        justify-content: center;
+        height: calculateRem(72px);
+        width: calculateRem(72px);
+        border-radius: 50%;
+        background: $ibexa-color-dark;
+
+        .ibexa-icon {
+            fill: $ibexa-color-white;
+        }
+    }
+
+    &__label {
+        font-family: $ibexa-font-family-headings;
+        font-weight: 600;
+        font-size: $ibexa-text-font-size-large;
+    }
+
+    &__value {
+        font-weight: 600;
+        font-size: $h1-font-size;
+        line-height: calculateRem(36px);
+    }
+
+    @mixin icon-colors($fill, $background) {
+        .ibexa-summary-tile {
+            &__icon-wrapper {
+                background: $background;
+            }
+        }
+
+        .ibexa-icon {
```

Comment:
I think this fragment should be inside `__icon-wrapper` modifier, as now you'll change each icon color, event the ones send from `value` parameter in twig